src/screens/DataApi/DataApi.tsx

```diff
@@ -36,6 +39,25 @@ export const sidebarSections: TocItem[] = [
   },
 ];
 
+const DataCoverageSection: React.FC<{}> = () => {
+  return (
+    <Grid container spacing={1}>
+      <Grid item xs={12}>
+        <DataCoverageTable rows={FIELDS_PRETTY_NAMES} />
+      </Grid>
+      <Grid item xs={12}>
+        <BlueLinkButton
+          trackingCategory={EventCategory.API}
+          trackingLabel="Data API: View all metrics"
+          href="https://apidocs.covidactnow.org/data-definitions"
+        >
```

Comment:
If you use `ExternalLink`, remember to call the tracking function in the `onClick` prop

```tsx
<ExternalLink href="..." onClick={() => trackViewAllMetrics()}>
  View more metrics
</ExternalLink>

```